wiki/Welcome/vi.md

```diff
@@ -0,0 +1,47 @@
+# Chào mừng
+
+Chào mừng đên osu!, một trò chơi nhịp điệu được lập trình bởi peppy với 4 chế độ chơi: osu!, chế độ bấm vòng tròn; osu!taiko, chế độ đánh trống; osu!catch, chế độ hứng quả; và osu!mania, chế độ bấm phím.
+
+## Một vài điều cơ bản
+
+- [Cài đặt](/wiki/Installation)
+- [Đăng kí](/wiki/Registration)
+- [Luật lệ](/wiki/Rules)
```

Comment:
```suggestion
- [Nội quy](/wiki/Rules)
```
Consistency with the directed article's title translations.
`Luật` means something like `law`, or rules of a country. Rules in a company, community, ... means `nội quy`, `quy định` in Vietnamese.